tests/pytorch/test_mpops.py

```diff
@@ -0,0 +1,195 @@
+import random
```

Comment:
1. Will this automatically triggered by CI?
2. I'd suggest to make the unittest file pattern have the strict correlation with the tested files. e.g. python/dgl/mpops/edgewise.py --> tests/python/dgl/mpops/test_edgewise.py

Reply:
Good idea. We can refactor the unit test frameworks altogether later.